cinch/roles/jenkins_slave/tasks/check_swarm_systemd.yml

```diff
@@ -0,0 +1,17 @@
+- name: check if swarm connected to the master with systemd
+  command: systemctl status swarm
+  ignore_errors: yes
```

Comment:
Can we go with "true" here, instead of "yes"? The rest of the codebase sticks with true/false instead of yes/no.